.editorconfig

```diff
@@ -0,0 +1,40 @@
+## EditorConfig is awesome: http://EditorConfig.org
+root = true
+
+
+## Newlines section
+
+# Make sure all files use lf as line ending
+[*]
+end_of_line = lf
+
+# Make sure our have an empty line at the end
+[*.{coffee,htaccess,html.twig,js,md,php}]
+insert_final_newline = true
+
+## Charset section
+[*.{coffee,htaccess,js,md,php,twig,xml,yml}]
+charset = utf-8
```

Comment:
Fixed in 798a9aa273cfc42d84505608bdfc2528d1bd9154

I tried to separate it by properties, and charset and newline are other properties